src/styles/components/_404.scss

```diff
@@ -0,0 +1,69 @@
+.error-page {
+  background-image: url("../../../public/images/background_image.png");
+  background-color: var(--background-color);
+  background-size: cover;
+  background-position: center;
+  padding: 0 1.25rem 1.25rem; // top: 0; left/bottom/right: 20px/16 (should mirror header padding)
+
+  @media (prefers-color-scheme: dark) {
+    background-image: url("../../../public/images/background_image_dark.svg");
+  }
```

Comment:
This kind of strays into personal preference territory, but I find the smaller 404 text difficult to read with the background image. On the homepage, any non-title text has a solid black or white background, which might be partly for legibility.